rbd/rbd.go

```diff
@@ -858,7 +865,10 @@ func (image *Image) WriteAt(data []byte, off int64) (n int, err error) {
 	return ret, err
 }
 
-// int rbd_flush(rbd_image_t image);
+// Flush all locally cached writes to storage.
```

Comment:
Not sure if it is only "local". Possibly writes have been cached at the OSD as well, and these would be flushed too.

Reply:
I read the ceph docs and that's what it implied. I can drop the word "locally" and leave it a bit ambiguous where the caching occurs if that would be OK.

Reply:
Yeah, that is fine. Dropping the "locally" really suggests it is up to Ceph to do their default (and there is no other choice).